client/src/ui/explore/OpportunityTimeline.tsx

```diff
@@ -2,69 +2,100 @@
+  interviewing: number
+  rejected: number
+}
+
+interface timelinePoints {
+  dateRangeStart: string
+  dateRangeEnd: string
+}
 export function OpportunityTimeline({ id }: oppornityDataProps) {
+
+  const [isClient, setIsClient] = useState(false);
+
+  useEffect(() => {
+    setIsClient(true);
+  }, []);
+
+  if (!isClient) {
+    return null; // Render nothing on the server side
+  }
   //future function for fetching timeline data
   const fetchTimelineData = (oppId: number) => {
     return console.log(oppId);
   };
 
   fetchTimelineData(id);
 
-  // Hard coded values for content card
-  const customContent = [
-    <div key={1} style={{ textAlign: "center" }}>
-      <Typography>Job Stats</Typography>
-      <Typography>Interviewing: 50</Typography>
-      <Typography>Rejected: 50</Typography>
-    </div>,
-    <div key={2} style={{ textAlign: "center" }}>
-      <Typography>Job Stats</Typography>
-      <Typography>Interviewing: 25</Typography>
-      <Typography>Rejected: 25</Typography>
-    </div>,
-    <div key={3} style={{ textAlign: "center" }}>
-      <Typography>Job Stats</Typography>
-      <Typography>Interviewing: 25</Typography>
-      <Typography>Rejected: 25</Typography>
-    </div>,
-    <div key={4} style={{ textAlign: "center" }}>
-      <Typography>Job Stats</Typography>
-      <Typography>Interviewing: 25</Typography>
-      <Typography>Rejected: 25</Typography>
-    </div>,
-    <div key={5} style={{ textAlign: "center" }}>
-      <Typography>Job Stats</Typography>
-      <Typography>Interviewing: 25</Typography>
-      <Typography>Rejected: 25</Typography>
-    </div>,
-  ];
-
-  //Hard coded Point titles
-  const items = [
+  const rstats: rangeStats [] = [
+    {
+      interviewing: 25,
+      rejected: 25
+    },
+    {
+      interviewing: 25,
+      rejected: 25
+    },
+    {
+      interviewing: 25,
+      rejected: 25
+    },
     {
-      title: <Typography color="white">04/29/24 - 05/7/24</Typography>,
+      interviewing: 25,
+      rejected: 25
     },
     {
-      title: <Typography color="white">05/8/24 - 05/15/24</Typography>,
+      interviewing: 25,
+      rejected: 25
+    },
+  ]
+
+  const tPoints: timelinePoints [] = [
+    {
+      dateRangeStart: "4/29/24",
+      dateRangeEnd: "5/7/24"
     },
     {
-      title: <Typography color="white">05/16/24 - 05/23/24</Typography>,
+      dateRangeStart: "4/29/24",
+      dateRangeEnd: "5/7/24"
     },
     {
-      title: <Typography color="white">05/24/24 - 05/30/24</Typography>,
+      dateRangeStart: "4/29/24",
+      dateRangeEnd: "5/7/24"
     },
     {
-      title: <Typography color="white">05/30/24 - 06/06/24</Typography>,
+      dateRangeStart: "4/29/24",
+      dateRangeEnd: "5/7/24"
     },
     {
-      title: <Typography color="white">05/30/24 - 06/06/24</Typography>,
+      dateRangeStart: "4/29/24",
+      dateRangeEnd: "5/7/24"
     },
-  ];
+  ]
+  // Hard coded values for content card
+  const customContent = 
```

Comment:
nit - I would make this into its own component rather than sitting on the same file as the OpportunityTimeline. Make the OpportunityTimeline component all about itself if possible